docs/example/saxml/README.md

```diff
@@ -0,0 +1,94 @@
+# Deploy SaxML Multihost with LWS on GKE
+
+Follow the steps on this [guide](https://cloud.google.com/kubernetes-engine/docs/tutorials/tpu-multihost-saxml#before-you-begin) upto the deploy saxml step
+to create GKE cluster, two TPU nodepools, and configure kubectl with the cluster. 
+
+## Install LeaderWorkerSet
+
+Follow the step-by-step guide on how to install LWS. [View installation guide](https://github.com/kubernetes-sigs/lws/blob/main/docs/setup/install.md)
+
+
+## Deploy ConfigMap with model configuration
```

Comment:
do you have an existing picture about how the deployments and each components look like and interact with each others?

Reply:
so users would understand why we are deploying this easier.

Reply:
Added picture